kayvee.go

```diff
@@ -39,6 +43,9 @@ func Format(data map[string]interface{}) string {
 	if deployEnv != "" {
 		data["deploy_env"] = deployEnv
 	}
+	if workflowID != "" {
+		data["workflow_id"] = workflowID
```

Comment:
We have a few systems that already log workflow_id and this would be overridden by them. Should we use ‘wm_workflow_id’ or ‘_workflow_id’ here or do you think we should disallow the use of workflow_id as a key when using workflows

Reply:
+1 - we could change the one logged in multiverse too. although, i do like the idea of renaming this to be something that's more clearly for workflow-manager workflows

Reply:
maybe `wf_id`?

Reply:
Changed in 97ea34c